plugins/sentinelone/plugin.spec.yaml

```diff
@@ -3044,6 +3044,29 @@ actions:
         type: integer
         required: true
         example: 1
+  move_between_sites:
+    title: Move Agent to Another Site
+    description: Move an agent to another Site, This command requires Account or Global level access for your user role
```

Comment:
Suggested reword: This command -> This action

Reply:
This has been updated